Services/UserService/Controllers/UsersController.cs

```diff
@@ -24,7 +28,15 @@ public UsersController([FromServices] ILogger<UsersController> logger)
         public async Task<bool> CreateUser([FromServices] ICreateUserCommand command, [FromBody] CreateUserRequest request)
         {
             logger.LogInformation("Create user request received from GUI to UserService");
-            return await command.Execute(request);
+            return await command.Execute(request); ;
+        }
+
+        [Route("confirm")]
+        [HttpGet]

```

Comment:
Why get, I think it's more of a post stuff or sth like that

Reply:
Because of follow link